src/Symfony/Component/Console/Command/LockableTrait.php

```diff
@@ -48,7 +48,7 @@ private function lock($name = null, $blocking = false)
             $store = new FlockStore();
         }
 
-        $this->lock = (new Factory($store))->createLock($name ?: $this->getName());
+        $this->lock = (new Factory($store))->createLock($name ?: $this->getName(), null);
```

Comment:
Related to this PR?

Reply:
Yes, it is. I didn't notice this until the build started failing and showed errors in `Console` component. With this change, when creating a lock using `Factory` we should override the `ttl` to be `null` for stores that don't support expiration because they throw the new exception.

Reply:
Reverted. See above.